package.json

```diff
@@ -34,8 +34,12 @@
   "author": "fe-app",
   "license": "MIT",
   "dependencies": {
-    "lit-html": "^1.0.0",
-    "lit-element": "^2.0.1"
+    "@lion/button": "^0.9.0",
```

Comment:
Change to 0.8.10, if normal import works means

Reply:
added and pushed the code